docs/UserGuide.adoc

```diff
@@ -18,25 +18,32 @@ By: `W13-1`      Since: `Sept 2018`      Licence: `MIT`
 
 == Introduction
 
-FreeTime is a desktop app that allows you to *find free timeslots to meet team members for projects*. More specifically, it helps you identify the times during which everyone you want to meet is free. Timetable Deconflicter is optimized for those who prefer to work with a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+_FreeTime_ is a desktop app that allows you to *find free timeslots to meet team members for projects*. More specifically, it helps you identify the times during which everyone you want to meet is free.
+_FreeTime_ is optimized for those who prefer to work with a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). Interested? Jump to the <<Quick Start>> to get started. Enjoy!
 
 == Quick Start
 
 .  Ensure you have Java version `9` or later installed in your Computer.
-.  Download the latest `addressbook.jar` link:{repoURL}/releases[here].
-.  Copy the file to the folder you want to use as the home folder for your Address Book.
+.  Download the latest `FreeTime.jar` link:{repoURL}/releases[here].
+.  Copy the file to the folder you want to use as the home folder for your application.
 .  Double-click the file to start the app. The GUI should appear in a few seconds.
+.  You should be on the FreeTime's login screen, as follows:
 +
-image::Ui.png[width="790"]
+.Window of FreeTime during first start-up
+image::StartUpScreen.png[width="790"]
 +
-.  Type the command in the command box and press kbd:[Enter] to execute it. +
-e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-.  Some example commands you can try:
+.  If it is the first time you are using the app, follow the instructions on-screen to create your account and press kbd:[Enter] to execute it. +
+****
+Creating account example: +
+
+`register Paul 123` +
```

Comment:
This is with consideration that the fields need not be all filled in for v1.4 @bennchong

Reply:
@nianfei97 @alexiscatnip. Just an update, for registering there will be prefixes to each argument, eg. register u/Paul pw/123. You can just push this first, I will make it a note to edit the UG as needed

Reply:
Noted with thanks